README.md

```diff
@@ -457,6 +494,43 @@ class DeliverViaSMSC
   end
 end
 ```
+
+### Custom Retry Queues
+
+By default, when a job is retried, it is added to the `@queue` specified in the worker. However, you may want to push the job into another (lower or higher priority) queue when the job fails. You can do this by dynamically specifying the retry queue. For example:
+
+```ruby
+class ExampleJob
+  extend Resque::Plugins::Retry
+  @queue = :testing
+  @retry_delay = 1
+
+  def self.work(user_id, user_mode, record_id)
+    Resque.enqueue_to(
+      target_queue_for_args(user_id, user_mode, record_id),
+      self,
+      user_id, user_mode, record_id
+     )
+  end
+
+  def self.retry_queue(exception, *args)
+    target_queue_for_args(*args)
+  end
+
+  def self.perform(*args)
+    raise
+  end
+
+  def self.target_queue_for_args(user_id, user_mode, record_id)
```

Comment:
The signature for this should probably be `*args` no?

Reply:
@jzaleski I think defining the arguments make this example a bit more clear. In this case, I'm demonstrating how you can use `retry_queue` to ensure that jobs from a test environment get thrown into a separate queue than production user jobs. IMHO `*args` would make the ex a bit more abstract and harder to grasp. Either way works for me though!